specification/hugr.md

```diff
@@ -615,7 +614,34 @@ Specifically, these rules allow for edges where in a given execution of
 the HUGR the source of the edge executes once, but the target may
 execute \>=0 times.
 
-<img src="attachments/2647818241/2647818338.png" width="768px">
+The diagram below is equivalent to the diagram in the [Dataflow](#dataflow)
+section above, but the input edge to "op3" has been replaced with a non-local
+edge from the surrounding DFG (the thick arrow).
```

Comment:
👍

Reply:
It's unfortunate that the two outputs from the nested graph also look like a double arrow! Maybe we should add another op which merges them into a single output for neatness?

Reply:
I'll try that.